### ISSUE TITLE
Coverage Questions

### ISSUE THREAD

**Issue by blongstreth** [sebakerckhof/stratosphere#10]
Hi:

I just stumbled into the project and excited about the potential.  Before I look into things further, I have some questions please:

1.  Does Stratosphere work with latest versions of Meteor (v1.4.2+)?
2.  Does Stratosphere cache the platform tools, etc.?  Meaning, if meteor.com and/or the packages site went offline and nothing is cached on the build machine, will the Meteor command line tool still work properly assuming all things are cached on Stratosphere.  Specifically interested in knowing if the platforms are cached as well.

In a nutshell, trying to assess if I can get Meteor to work locally without Meteor.com access.

Thanks in advance,

Bradley

**Comment by sebakerckhof** [comment]
@blongstreth 

1. Yes
2. Partially. What is cached is the database. The database basically contains all metadata of packages, versions, builds, ...  However, the actual files itself are stored on amazon S3 and are not cached by stratosphere. One could extend stratosphere to do this, by modifying the synchronization process a bit.
Now stratosphere synchronizes with the meteor package servers. In this process, you could change the URL's of versions/builds to pass via stratosphere and cache the packages if they are requested.

But this is not a goal of stratosphere. It's mainly about making private packages work (which are stored by stratosphere).

However, in my development team this was also a concern and we basically backup the local cache of the meteor packages on our build machines.
Every package used by a meteor application on your computer is downloaded to a package cache in the meteor installation location. On our build machines this package cache is back-upped in case Meteor.com would go offline.

**Comment by blongstreth** [comment]
Thanks for the quick reply!  We use Artifactory for all of our stuff except Meteor.  Long term, my plan is to convert to npm so I won't have to worry about this.  For now, I need something that is easy, robust and allows for developers to build locally on their dev. boxes (which can be mac, linux or windows).  With that in mind which approach would you take?  I too have been backing up the local cache on build machines under linux but need to consider other contexts.

Thanks in advance.
